src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -505,6 +506,49 @@ private fun RsMod.insertUseItem(psiFactory: RsPsiFactory, usePath: String) {
     }
 }
 
+private fun RsMod.tryMergeWithOtherUseItems(psiFactory: RsPsiFactory, newUseItem: RsUseItem): Boolean =
+    childrenOfType<RsUseItem>().any { newUseItem.tryMergeWith(psiFactory, it) }
+
+private fun RsUseItem.tryMergeWith(psiFactory: RsPsiFactory, other: RsUseItem): Boolean {
+    if (other.vis != null || other.outerAttrList.isNotEmpty() || other.useSpeck?.isStarImport == true) return false
+
+    val parentPath = parentPath ?: return false
+    val importingName = importingNames?.singleOrNull() ?: return false
```

Comment:
parentPath and importingName is computed multiple times for "this" when could be computed once